doc/asciidoc/cud-file-format/sink-cud-template.adoc

```diff
@@ -0,0 +1 @@
+{environment}.topic.cud=<LIST_OF_TOPICS_SEPARATED   _BY_SEMICOLON>
```

Comment:
extra spaces after SEPARATED render in a way that's broken up

Reply:
I'll fix it and i'll update the PR. Thanks for pointing that out